static/app/views/projectDetail/projectDetail.tsx

```diff
@@ -197,6 +201,12 @@ class ProjectDetail extends AsyncView<Props, State> {
         disableMultipleProjectSelection
         skipLoadLastUsed
         onUpdateProjects={this.handleProjectChange}
+        relativeDateOptions={
+          hasOnlyBasicChart
+            ? pick(DEFAULT_RELATIVE_PERIODS, ['1h', '24h', '7d', '14d', '30d'])
```

Comment:
can we maybe reuse this const? https://github.com/getsentry/sentry/blob/6d63a529195dc5da9e606c5de97de4f1f2e9f0de/static/app/views/projectDetail/charts/projectErrorsBasicChart.tsx#L15